### ISSUE TITLE
Move this into CRS

### ISSUE THREAD

**Issue by samsonasu** [tenforwardconsulting/ssl_expr#1]
This doesn't need to be it's own repo, it should probably just be a method somewhere in CRS.  On the beta branch there's a bunch of new stuff with SSL certificates and it wouldn't be hard to find a logical place.

**Comment by petitJAM** [comment]
Yea I just wanted to put it somewhere other than my computer.

**Comment by samsonasu** [comment]
Yeah I figured.  This works well to and I think you can make them private: https://gist.github.com/
